apps/backend-sls/prisma/seed.ts

```diff
@@ -282,12 +282,62 @@ async function main(prisma: Prisma.PrismaClient) {
     update: {},
   })
 
+  const question6 = await prisma.question.upsert({
+    where: {
+      id: 5,
+    },
+    create: {
+      id: 5,
+      name: 'Numerische Testfrage',
+      content: 'Was ist richtig?',
+      contentPlain: 'Was ist richtig?',
+      type: 'NUMERICAL',
+      options: {
+        restrictions: {
+          min: 0,
+        },
+        solutionRanges: [
```

Comment:
Solutions for numerical questions are defined in ranges. A range with only min is exact, meaning that it will only match the exact number. Questions don't need to have a solution (optional), but without solution, they will not be usable for gamified features.